charts/alfresco-search-enterprise/templates/deployment.yaml

```diff
@@ -15,9 +15,6 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/secret-broker: {{ include (print $.Template.BasePath "/secret-messagebroker.yaml") $ | sha256sum }}
-        checksum/secret-database: {{ include (print $.Template.BasePath "/secret-database.yaml") $ | sha256sum }}
-        checksum/secret-elasticsearch: {{ include (print $.Template.BasePath "/secret-elasticsearch.yaml") $ | sha256sum }}
```

Comment:
keep them, we know they are sub-optimal but having them improve user experience for a subset of use cases and no downsides when using external secrets

Reply:
I mostly removed them because they make the tests less DRY as I would be forced to include the secrets tenplate too and the `documentSelector` feature is not flexible enough to support multiple conditions.
Plus, the checksum mecanism is not present in the mediation statefulset so the component would anyway not work after a CM update